skiko/src/nativeJsMain/cpp/common.h

```diff
@@ -126,16 +126,15 @@ KNativePointer ptrToInterop(T* ptr) {
     return ptr;
 }
 
-#ifdef __clang__
-__attribute__((noreturn))
-void TODO(const char*);
-__attribute__((noreturn))
-void SKIKO_ASSERT(bool, const char*);
+#if defined(__clang__) || defined(_MSC_VER)
```

Comment:
Guess for _MSC_VER we shall use `__declspec(noreturn)`, it's Visual Studio compiler which didn't know clang attributes, IIRC.